core/controllers/admin_test.py

```diff
@@ -113,6 +93,16 @@ def test_admin_page_rights(self):
         self.get_html_response('/admin')
         self.logout()
 
+    def test_promo_bar_configuration_not_present_to_admin(self):
+        """Test that promo bar configuration is not presentd in admin page."""
+        self.login(self.ADMIN_EMAIL, is_super_admin=True)
+
+        response_dict = self.get_json('/adminhandler')
+        response_config_properties = response_dict['config_properties']
+
+        self.assertNotIn('promo_bar_enabled', response_config_properties)
```

Comment:
For completeness, pick some standard config property and do an assertIn on it in exactly the same way.

Reply:
Done!